src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
 public class Application {
+    static int BALL_COUNT = 3;
```

Comment:
The constant `BALL_COUNT` should be named according to the Java style guide for constants. Consider renaming it to `BALL_COUNT` to follow the convention of using uppercase letters with underscores for constant names .